youtube_dl/extractor/nbc.py

```diff
@@ -50,6 +50,56 @@ def _real_extract(self, url):
         return self.url_result(theplatform_url)
 
 
+class NBCSportsVPlayerIE(InfoExtractor):
+    _VALID_URL = r'https?://vplayer\.nbcsports\.com/(?:[^/]+/)+(?P<id>[0-9a-zA-Z]+)'
```

Comment:
Looks like `_` should be allowed for `id` (e.g. http://vplayer.nbcsports.com/p/BxmELC/nbc_embedshare/select/_hqLjQ95yx8Z)